app-react/src/jsMain/kotlin/com/ustadmobile/mui/components/SocialWarningListItem.kt

```diff
@@ -0,0 +1,87 @@
+package com.ustadmobile.mui.components
+
+import com.ustadmobile.core.MR
+import com.ustadmobile.core.hooks.useStringProvider
+import mui.icons.material.Circle
+import mui.icons.material.Warning
```

Comment:
As per note on app-react docs: import of icons must use the as - otherwise IDE performance is extremely poor. e.g. use
```
import mui.icons.material.Circle as CircleIcon
```